brick.js

```diff
@@ -0,0 +1,22 @@
+class Brick {
+  constructor(ctx, x, y, width, height, color) {
+    this.ctx = ctx;
+    this.x = x;
+    this.y = y;
+    this.width = width;
+    this.height = height;
+    this.color = color;
+    this.top = this.y;
```

Comment:
whats this for? you don't use this in the game. would just make these functions to return these values if you really need it